formats/table/attributors/custom_attributor_prefix.js

```diff
@@ -0,0 +1,47 @@
+import { TABLE_ATTRIBUTES, TABLE_STYLES, tableConfig } from './table_config';
+import {
+  cellConfig,
+  TABLE_CELL_ATTRIBUTES,
+  TABLE_CELL_STYLES,
+} from './cell_config';
+
+const buildTagInfo = (name, keyNamesSet) => (result, tag) => {
+  result[tag] = {
+    name,
+    keyNamesSet,
+  };
+  return result;
+};
+
+const CUSTOM_ATTRIBUTERS_TAG_INFO = {
```

Comment:
Why custom? Initially all attributors are custom. Maybe TABLE_CELL?

Reply:
Discussed.
Renamed to overridden.